### PR TITLE
Clean code

### PR DESCRIPTION
**Pull request by yabirgb**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: rotkehlchen/chain/gnosis/modules/airdrops/decoder.py [hunk 1]
```diff
@@ -1,36 +1,17 @@
...
 logger = logging.getLogger(__name__)
 log = RotkehlchenLogsAdapter(logger)
 
 
 class AirdropsDecoder(DecoderInterface):
 
-    def __init__(
-            self,
-            ethereum_inquirer: 'EthereumInquirer',
-            base_tools: 'BaseDecoderTools',
```
Comment (yabirgb) [inline rotkehlchen/chain/gnosis/modules/airdrops/decoder.py:23]: should we just remove this and move it to cowswap?

Comment (yabirgb) [reply rotkehlchen/chain/gnosis/modules/airdrops/decoder.py:23]: This doesn't seem to be useful. Removing it

File: rotkehlchen/chain/gnosis/modules/airdrops/decoder.py [hunk 2]
```diff
@@ -1,36 +0,0 @@
-import logging
```
Comment (LefterisJP) [inline rotkehlchen/chain/gnosis/modules/airdrops/decoder.py:1]: Hmm so it seems this was an empty decoder class? Right?

**Review by LefterisJP — APPROVED** [review]
LGTM

File: rotkehlchen/chain/gnosis/modules/airdrops/decoder.py (hunk 2 above)
Comment (yabirgb) [reply rotkehlchen/chain/gnosis/modules/airdrops/decoder.py:1]: yes, the only thing it was doing was to add the cowswap counterparty but we already have it with the decoder for the swaps. This one was a no-op it seems

**Pull request merged**